fitnessplatform/urls.py

```diff
@@ -24,7 +24,8 @@
 urlpatterns = [
     path('', start_view),
     path('admin/', admin.site.urls),
-    path('user/', include('allauth.urls')),
+    #path('user/', include('allauth.urls')),
+    path('accounts/', include('allauth.urls')),
     path('user/', user_profile, name='user_profile'),
```

Comment:
Sollten wir hier den Pfad auch in `accounts/` ändern? Dann wäre es konsistenter...